.travis.yml

```diff
@@ -2,78 +2,69 @@ os: linux
 language: scala
 env:
   global:
-    - AKKA_TEST_TIMEFACTOR=10
-    - SBT_TASK_LIMIT=4
-    - SCALA_VERSION=2.12.4
+  - AKKA_TEST_TIMEFACTOR=10
+  - SBT_TASK_LIMIT=4
+  - SCALA_VERSION=2.12.4
+  matrix:
+  - secure: cOdXmYMqs8y1qVFjdLq68QAkLnU1+o4Cqjo7C63xMaj94bGgcc+fncRaVu/R+FTVymuFPGe2TyCmAkzEHn1EAWPRkvXftQgAdlBXYCYdZ7E+d7I6kX0gELZHubtipq8rCrookAtAnqz1jOBYrHbJjaDlvw6UI1WAS95y/hIZmHo=
+  - secure: N2ln4PPhdM+qTpL9FP158QUQaGiI+H1WtjmsNgsABJhmbT3NCyR+hcsVs+/gJDctFsUItATAsp9rDsVqhvNixQ7F0gcC45MkjzYLuez01K6YGjMNr8Rse7GlFxRsFYDhOboxRSgCML5LAVBQoh1LitYb4VxtHeQytqAr5MVrhXM=
 before_install:
-  - if [[ "$TRAVIS_OS_NAME" = "osx" ]]; then
-      brew update;
-      brew install sbt;
-      export JAVA_HOME=$(/usr/libexec/java_home);
-    fi
+- if [[ "$TRAVIS_OS_NAME" = "osx" ]]; then brew update; brew install sbt; export JAVA_HOME=$(/usr/libexec/java_home);
+  fi
 stages:
-  - name: test
-  - name: integration
-  - name: publish
-    if: branch = master
```

Comment:
actually this is probably a mistake I introduced as ensime does not have a master branch

Reply:
(should be 3.0)

Reply:
ah, I can fix that here, well spotted